### ISSUE TITLE
KDE version?

### ISSUE THREAD

**Issue by Akrai** [protesilaos/tempus-themes#1]
Hi, could you please add a kde version of the colors?

**Comment by protesilaos** [comment]
This is being worked on. I still need to finalise the templates. Will post an update here once they are ready.

**Comment by protesilaos** [comment]
Added support for Konsole (KDE terminal). The files are in this repo's [konsole directory](https://github.com/protesilaos/tempus-themes/tree/master/konsole).

I will provide an AUR package based on the [tempus-themes-konsole](https://github.com/protesilaos/tempus-themes-konsole) ~~in few hours~~. Here is the [AUR package](https://aur.archlinux.org/packages/tempus-themes-konsole/).

More KDE-related stuff will follow.

**Comment by protesilaos** [comment]
Just to update on this. Here is what I have learnt thus far.

**tl;dr** KDE is complex. More testing is needed.

KDE has a rather intricate way of defining colour schemes and syntax themes. Kate/Kwrite have their own schemas. Meanwhile, KDE Plasma provides support for custom colour schemes which can be used by a wide variety of applications, including the default Breeze theme. Furthermore, there appear to be plans for refocusing development on Qt syntax highlighting, which would help simplify things. See, for example, [this Reddit thread](https://www.reddit.com/r/kde/comments/6mvsvm/where_can_one_find_kate_color_schemas/).

Notwithstanding the Konsole themes I published earlier today (see my previous comment in this thread), I have also developed an experimental template for KDE Plasma colour schemes. It is available for testing in the [`experimental` branch of the tempus-themes-generator](https://github.com/protesilaos/tempus-themes-generator/blob/experimental/templates/plasma-color-scheme). **Note for anyone interested:** any generated file should have a `.colors` file type extension and should be placed under `~/.local/share/color-schemes/`.

While I appreciate the idea of an overarching colour scheme for the desktop environment, my impression so far is that my themes are not suitable for general UI elements. I would even argue that even the default KDE Breeze theme has certain issues with colour contrast, such as 'selection' being blue and certain elements being a slightly different blue. But I digress.

For the time being, the tests are inconclusive. I will explore the possibilities further. Based on my current understanding though, I do not think I will be porting over any of my existing projects. At least not without considerable adjustments.
